plugins/scaffolder-node/report.api.md

```diff
@@ -493,6 +495,25 @@ export interface TaskContext {
 // @public
 export type TaskEventType = 'completion' | 'log' | 'cancelled' | 'recovered';
 
+// @public
+export type TaskFilter = {
```

Comment:
If possible it would be best to leave these as internal types, but that might not be possible seeing as `TaskBroker` is exported